README.md

```diff
@@ -0,0 +1,125 @@
+<a name="readme-top"></a>
+  <h3><b>Recipe App</b></h3>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [Catalog of my things] <a name="about-project"></a>
+
+**[Catalog of my things]** Blog App is a web application that allows users to create, read, update, and delete blog posts. Users can sign up for an account, create new blog posts, and edit or delete their existing posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">Ruby on rails</a></li>
+  </ul>
+
+  <summary>Database</summary>
+  <ul>
+    <li><a href="#">postgreSQL</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+
+- **[Add new post]**
+- **[Add new comment]**
+- **[Add new like]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Preriquisites
+
+Ruby version >= 3.0.0
+
+### Setup
+
+Clone this repository to your desired folder:
+sh
+  cd my-folder
+ git clone https://github.com/ernestmusong/Blog-app.git
+
+ ### Install:
+ Run: bundle install
+ Run: npm install
+
+ ### Start:
+ Run: rails s
+
+ ### Tests:
+ Run: rspec
+
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Emmanuel Obi**
+- GitHub: [@Gmanlove](https://github.com/Gmanlove)
+
```

Comment:
- [OPTIONAL] It would be better if the other author add more contact information like LinkedIn and twitter 😁